glue/sales_val_flagging.py

```diff
@@ -102,12 +102,35 @@ def ptax_adjustment(df, groups, ptax_sd):
 
     group_string = "_".join(groups)
 
-    df["ptax_flag_w_deviation"] = df["ptax_flag_original"] & (
-        (df[f"sv_price_deviation_{group_string}"] >= ptax_sd[1])
-        | (df[f"sv_price_deviation_{group_string}"] <= -ptax_sd[0])
-        | (df[f"sv_price_per_sqft_deviation_{group_string}"] >= ptax_sd[1])
-        | (df[f"sv_price_per_sqft_deviation_{group_string}"] <= -ptax_sd[0])
-    )
+    if condos == False:
+        df["ptax_flag_w_deviation"] = df["ptax_flag_original"] & (
+            (df[f"sv_price_deviation_{group_string}"] >= ptax_sd[1])
+            | (df[f"sv_price_deviation_{group_string}"] <= -ptax_sd[0])
+            | (df[f"sv_price_per_sqft_deviation_{group_string}"] >= ptax_sd[1])
+            | (df[f"sv_price_per_sqft_deviation_{group_string}"] <= -ptax_sd[0])
+        )
+
+        # Determine the ptax direction
+        conditions = [
+            (df[f"sv_price_deviation_{group_string}"] >= ptax_sd[1])
+            | (df[f"sv_price_per_sqft_deviation_{group_string}"] >= ptax_sd[1]),
+            (df[f"sv_price_deviation_{group_string}"] <= -ptax_sd[0])
+            | (df[f"sv_price_per_sqft_deviation_{group_string}"] <= -ptax_sd[0]),
+        ]
+    else:
+        df["ptax_flag_w_deviation"] = df["ptax_flag_original"] & (
+            (df[f"sv_price_deviation_{group_string}"] >= ptax_sd[1])
+            | (df[f"sv_price_deviation_{group_string}"] <= -ptax_sd[0])
+        )
+
+        # Determine the ptax direction
+        conditions = [
+            (df[f"sv_price_deviation_{group_string}"] >= ptax_sd[1]),
+            (df[f"sv_price_deviation_{group_string}"] <= -ptax_sd[0]),
+        ]
+
+    directions = ["High", "Low"]
+    df["ptax_direction"] = np.select(conditions, directions, default=np.nan)
```

Comment:
I added this when working with @ccao-jardine. It splits PTAX by high or low outlier in `sv_outlier_type`. Nothing functionally changes, but one more value type will be added in `sv_outlier_type`. I added it to this PR because I think it makes sense to include as we are probably going to have to do a full re-run anyway to re-flag these condos, even if we don't go with a overall methodology change.